baseline.yaml

```diff
@@ -649,6 +649,30 @@ criteria:
     security_insights_value: # TODO
     scorecard_probe: # TODO
 
+- id: OSPS-DO-14
+    maturity_level: 3
+    category: Documentation
+    criteria: |
+       The project documentation MUST provide a 
+       descriptive statement when releases or 
+       versions are no longer supported and that 
+       will no longer receive security updates.  
+       
+       This should be provided both in human and 
+       machine-readable formats. 
+    objective: |
+       To advertise to downstream consumers when software 
+       has reached "End of X" [Support, Life, Maintenance, etc.].
+    implementation: |
+       Create a status check that checks the project's
+       version control system for support and/or lifecycle
+       statements.  Publishing this in machine-readble 
+       formats is preferable.
```

Comment:
The "implementation" sections should tell projects how to implement it, *not* how to determine if it's true or not.

Maybe we need a new field, similar to the Best Practices badge's "autofill" field, to describe how a criterion could be automatically determined. Note that the Best Practices Badge has a "details" field that explains how *projects* can meet the criterion, and I believe that this is what the current "implementation" field is for.

Reply:
I'd _like_ for the implementation guidance to projects to be clear enough that someone defining automation can simply read the implementation description to determine how to check it.  For example, "Publish support information in $FORMAT_X or $FORMAT_Y, with a link at $LOCATION to the support document."